openshift/ftest-openshift-assistant-template/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>arquillian-cube-openshift-parent</artifactId>
+    <groupId>org.arquillian.cube</groupId>
+    <version>1.12.1-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>arquillian-cube-openshift-assistant-template-ftest</artifactId>
```

Comment:
Makes we wonder if we really need a new module for that. Shouldn't we just add it to `ftest-openshift-assistant`? WDYT @lordofthejars

Reply:
Also, if we keep it this way we shouldn't release it to maven central - there is probably no point for anyone to consume `ftest` jars.

Reply:
> Shouldn't we just add it to ftest-openshift-assistant? 

I tried this initially, but that module already contains an openshift.json file, which is read by default from the classpath, though it gets overridden by the template we apply. But the output is kind of confusing, it first logs something like, `Reading from template file, no external resources found` etc., which I found quite misleading at the start.  

@bartoszmajsak, @lordofthejars - would like to know the best and most optimized way of handling this.

Reply:
Yes Hemani, you are right, leave it as a separate ftest. Basically the `openshift-assistant` ftest is testing that the default cube parameters also work with `OpenShiftAssistant` so yeah.

Regarding ftest modules, I think that for now there are some of them that are released to Maven central, so let's fix this problem as well

Reply:
Can't we use another filename though for this test?

Reply:
This still doesn't help, because classpath is searched for configuration file first. And the output is as follows:

``` 
Applying kubernetes configuration from: file:/home/hemani/trello/arquillian-cube/openshift/ftest-openshift-assistant/target/test-classes/hello-openshift.json
Processing template. No parameters file has been specified, processing without external parameters!
```

Reply:
@lordofthejars Is this a bug then?

Reply:
No it is not a bug, by default arquillian "kube" loads some defaults- And this is what I wanted to test in that ftest, that defaults are loaded independently of openshiftassistant.